### PR TITLE
store_metadata: if pulp was used, only report pulp repositories

### PR DESCRIPTION
**Pull request by twaugh**
Otherwise the temporary Distribution pushes are also reported, which we don't want.

cf. koji_import

### CODE REVIEW

File: atomic_reactor/plugins/exit_store_metadata_in_osv3.py [hunk 1]
```diff
@@ -72,19 +87,26 @@ def get_repositories(self):
             "unique": unique_repositories,
         }
 
-    def get_digests(self):
+    def get_pullspecs(self, digests):
+        if self.workflow.push_conf.pulp_registries:
```
Comment (TomasTomecek) [inline atomic_reactor/plugins/exit_store_metadata_in_osv3.py]: This means, that it won't be possible to configure deployment when using distribution and pulp, both as final registries. I think it's totally fine.

**Comment by TomasTomecek** [comment]
LGTM, very nice catch Tim!

File: atomic_reactor/plugins/exit_store_metadata_in_osv3.py (hunk 1 above)
Comment (twaugh) [inline atomic_reactor/plugins/exit_store_metadata_in_osv3.py]: Right. If someone needs this in the future we can make it smarter and match up the dockpulp environment name in config.

**Comment by mmilata** [comment]
LGTM!

**Pull request merged**